Explorer/Assets/Scripts/Global/Dynamic/DynamicWorldContainer.cs

```diff
@@ -294,11 +297,24 @@ async UniTask InitializeContainersAsync(IPluginSettingsContainer settingsContain
                 exposedGlobalDataContainer.CameraSamplingData
             );
 
-            container.UserInAppInitializationFlow = new RealUserInitializationFlowController(
+            var livekitHealthCheck = bootstrapContainer.DebugSettings.EnableEmulateNoLivekitConnection
+                ? new IHealthCheck.AlwaysFails("Livekit connection is in debug, always fail mode")
+                : new SeveralHealthCheck(
+                      new MultipleURLHealthCheck(staticContainer.WebRequestsContainer.WebRequestController, bootstrapContainer.DecentralandUrlsSource,
```

Comment:
It's just a shortcut to don't write a construction with `ParallelHealthCheck` and manually put URLHealthCheck there every time. It makes a code more readable on the upper level

Reply:
On this line of question, do you think the `Decorator` flexibility is worth it here? 
I wonder if just one class could handle this requirement, in which you pass a list of `IHealthChecks` (sequential and parallel), as well as handling the retries (which don't seem optional to me). 
WDYT?

Reply:
Should we consider simplicity here over flexibility?

Reply:
Why do see a complexity if each object is small and cohesive? It's possible to mix it into a single object and grow it with multiple concerns, but what's benefits of it? SRP principle

It's mainly builder pattern, or is it a complexity for an example in [ASP.net](http://asp.net/) style?

```
builder.Services.AddControllers();
builder.Services.AddEndpointsApiExplorer();
builder.Services.AddSwaggerGen();
```

You can take a look for IStartupOperation, if I would bring new concerns into RealUserInAppInitializationFlow it would grow it and make it extra complex to maintain with many operational methods. I want to avoid this complexity with small, simple and cohesive objects. WDYT?

Reply:
I was thinking of the scattered logic for validating the `IHealthChecks`.  I see the decorator pattern as a flexible one, which would let me further expand the wrappers around concrete actions (in this case, validating the URLs). I can't imagine further decorators in this case, and maintaining a decorator is more complex than maintaining just a single class holding the simple logic. In this particular case, I wouldn't use this pattern.
On the other hand, I agree that this decorator follows SRP nicely, operations are concise inside of them. Although I dont fully agree on using it here, I can see the intention. And given that the logic involved is not that complex, we can move forward with it